pkg/slack/slack.go

```diff
@@ -0,0 +1,35 @@
+package slack
+
+import (
+	"fmt"
+	"github.com/navikt/nada-backend/pkg/graph/models"
+	"github.com/sirupsen/logrus"
+	"github.com/slack-go/slack"
+)
+
+type SlackClient struct {
+	log            *logrus.Logger
+	webhookurl     string
+	datakatalogurl string
+}
+
+func NewSlackClient(log *logrus.Logger, webhookurl string, datakatalogurl string) *SlackClient {
+	return &SlackClient{
+		log:            log,
+		webhookurl:     webhookurl,
+		datakatalogurl: datakatalogurl,
+	}
+}
+func (s SlackClient) NewDataProdukt(dp *models.Dataproduct) error {
```

Comment:
```suggestion
func (s SlackClient) NewDataproduct(dp *models.Dataproduct) error {
```

Holde koden til engelsk :)

Reply:
skal fikse. "Innafor" er innafor for meg 😄